docs/api/effector/sample.md

```diff
@@ -0,0 +1,197 @@
+---
+id: sample
+title: sample
+hide_title: true
+---
+
+# `sample(sourceStore, clockEvent, fn)`
+Overall this method can be used in order to link two nodes, resulting the third one, which will fire only upon `clock` node trigger.  
+_Note: [_`Unit`_](Unit.md) chapter may come in handy, to be aware of way of using this method._
+
+Passes current `sourceStore`'s state and `clockEvent`'s value to `fn` handler. Quite a common case when you need to handle some event with some store's state. Instead of using `store.getState()`, in body of effect, which may cause race conditions and inconsistency of state at the moment of effect thunk invocation.
+
+#### Arguments
+
+1. `sourceStore` _(Store)_: Source event
+2. `clockEvent` _(Event)_: Clock(Trigger) event
+3. `fn`? _(Function)_: Callable handler (may be [_`Event`_](Event.md) or [_`Effect`_](Effect.md) aswell), should be **pure**
+
+#### Returns
+
+([_`Event`_](Event.md)) - Event, which fires upon clock is triggered
+
+#### Example
+
+```js
+const clock = createEvent();
+const inc = createEvent();
+const store = createStore(0)
+  .on(inc, (state) => state + 1);
+
+const sampled = sample(store, clock, (i) => `Current count is ${i}`);
```

Comment:
if we don't use the second argument of the handler, then there is no need for it, since the source value will be passed on by default and we can use `.map` as usual